pkgs/development/mobile/genymotion/default.nix

```diff
@@ -67,7 +75,7 @@ stdenv.mkDerivation rec {
      '';
     homepage = https://www.genymotion.com/;
     license = stdenv.lib.licenses.unfree;
-    platforms = stdenv.lib.platforms.linux;
+    platforms = ["x86_64-linux"];
```

Comment:
The 64-bit version is downloaded above:

    name = "genymotion-${version}-linux_x64.bin";